### ISSUE TITLE
Conversation for Action

### ISSUE THREAD

**Issue by fosterlynn** [w3c-social/social-vocab#14]
The question is if the Conversation for Action protocol is an appropriate area to explore for this group.  If so, I will pull together more background info, and try to find vocabs with overlap with this concept (I'm sure there are a number of them).  (Examples of potential parts of this vocab would be: Offer, Request, Counter, Reject, Withdraw, Accept.)

Background: The term "Conversation for Action" comes from the book _Understanding Computers and Cognition_ by Terry Winograd and Fernando Flores, but very similar message patterns are used in every EDI standard. It is basically a structured computerized protocol for agreements between independent agents.  Of course, there have been Offer and Acceptance patterns going way back in business history, and legal contracts can come out of this interaction.  

But it would be good to be more broad in the social context, basically coming to agreements on collaborative actions, including but not limited to business use.  And it would need to allow more than 2 participants in the conversation.

What do people think of this group going down this path?

cc: @bhaugen (who has done a lot more work than me on CfA)

**Comment by akuckartz** [comment]
Can you provide a link or two ?

**Comment by fosterlynn** [comment]
@akuckartz These are all over the place, but together may provide some idea:
http://conversationsforaction.com/cfa-playground
http://www.itu.dk/people/kasper/REA2004/pospapers/PrasadJayaweera.pdf
http://en.wikipedia.org/wiki/Offer_and_acceptance
https://github.com/HeleneFi/The_Project/blob/master/Conversation.to.Action-Pull.Platform.md

Here's a discussion about it, where I got the links in the first place:
https://www.loomio.org/d/69tuWRqB/conversation-for-action

TL;DR: Here's an (incomplete) picture that might help:
![cfa_state_diagram_2](https://cloud.githubusercontent.com/assets/3776081/8089666/f1f91b26-0f6e-11e5-83d6-838c5e514148.png)

**Comment by fosterlynn** [comment]
A couple more links:
http://conversationsforaction.com/history/basic-action-workflow
http://www.ebxml.org/specs/bpPATT_print.pdf pg 23
